docs/website/docs/command-reference/delete.md

```diff
@@ -0,0 +1,38 @@
+---
+title: odo delete
+sidebar_position: 2
+---
+
```

Comment:
Could you find a presentation sentence to start the document, before entering the first paragraph?